HOWTO.md

```diff
@@ -0,0 +1,109 @@
+# Changes in recent Tigervnc versions
+Previous Tigervnc versions had a wrapper script called `vncserver`. This script
+could be run as a user manually to start *Xvnc* process. The usage was quite 
+simple as you just run:
+```
+$ vncserver :x [vncserver options] [Xvnc options]
+```
+and that was it. It worked just fine, however, there were issues when users
+wanted to use it in combination with *systemd*. Therefore, the implementation
+had to be changed to comply with *SELinux* and *systemd* rules.
+
+# How to start Tigervnc server
+## Add a user mapping
+With this you can map a user to a particular port. The mapping should be done in 
+`/etc/tigervnc/vncserver.users` configuration file. It should be pretty 
+straightforward. Once you open the file you will see there are some examples, but basically
+the mapping is in form:
+```
+:x=user
+```
+For example you can have
+```
+:1=test
+:2=vncuser
+```
+
+## Configure Xvnc options
+To configure Xvnc parameters, you need to go to the same directory where you did
+the user mapping and open `vncserver-config-defaults` configuration file. This 
+file is for the default Xvnc configuration and will be applied to every user 
+unless any of the following applies:
+* The user has its own configuration in `$HOME/.vnc/config`.
+* The same option with different value is configured in 
+  `vncserver-config-mandatory` configuration file, which replaces the default 
+  configuration and has even a higher priority than the per-user configuration.
+  This option is for system administrators when they want to force particular 
+  *Xvnc* options.
+
+Format of the configuration file is also quite simple as the configuration is
+in form of:
+```
+option=value
+option
+```
+for example:
+```
+session=gnome
+securitytypes=vncauth,tlsvnc
+desktop=sandbox
+geometry=2000x1200
+localhost
+alwaysshared
+```
+### Note:
+There is one important option you need to set and that option is the session you
+want to start. E.g when you want to start GNOME desktop, then you have to use:
+```
+session=gnome
+```
+This should match the name of a session desktop file from `/usr/share/xsessions`
+directory.
```

Comment:
This setting is right now optional as it will pick the first desktop it finds.

Reply:
It is optional, but I would still recommend to specify it, because the very first desktop might not work correctly. This situation for example happens on RHEL 8 with Kiosk mode installed.